tests/cephadm/test_label_no_schedule.py

```diff
@@ -0,0 +1,73 @@
+from json import loads
+
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class LabelNoScheduleError(Exception):
+    pass
+
+
+def adding_label(instance, node):
+    """
+    Method add _no_schedule label in the node
+    Args:
+        instance: cluster instance
+        node: cluster node
+    Returns:
+        boolean
+    """
+    log.info(f"Adding '__no_schedule' label in {node}")
+    cmd = f"cephadm shell ceph orch host label add {node} _no_schedule"
```

Comment:
Instead try to use method from https://github.com/red-hat-storage/cephci/pull/1833/files#diff-775b7df2434d2d775287b5a432b25dbdabedf3db60be80982b743d2fb8f5a400

Reply:
Updated